examples_tests/39.DenoiserTonemapper/ShaderCommon.glsl

```diff
@@ -8,6 +8,8 @@ layout(local_size_x=COMPUTE_WG_SIZE) in;
 layout(constant_id = 1) const uint EII_COLOR = 0u;
 layout(constant_id = 2) const uint EII_ALBEDO = 1u;
 layout(constant_id = 3) const uint EII_NORMAL = 2u;
+// layout(constant_id = 4) const uint EII_COUNT = 3u; // for now until @Crisspl fixes a bug with SPIRV-Cross usage
+#define EII_COUNT 3u
```

Comment:
@Crisspl new bug for you

Reply:
spec constant as descriptor count still works in ex29, you must be doing something wrong. What are the "symptoms"?

Reply:
it stopped throwing errors after we fixed the layout bidning's array counts
https://github.com/Devsh-Graphics-Programming/Nabla/pull/84/commits/88563ab81083d350ec9ddb3e8b709579c1e697b4